op-chain-ops/genesis/setters.go

```diff
@@ -147,3 +147,14 @@ func MigrateDepositHashes(hh *hardhat.Hardhat, db vm.StateDB) error {
 	})
 	return nil
 }
+
+// SetPrecompileBalances will set a single wei at each precompile address.
+// This is an optimization to make calling them cheaper. This should only
+// be used for devnets.
+func SetPrecompileBalances(db vm.StateDB) {
+	for i := 0; i <= 9; i++ {
```

Comment:
Can we do this for the first 256 addresses? This is what Geth does during genesis setup: https://github.com/ethereum/go-ethereum/blob/4c114af5024943a7471aeccdf5e536584c1e21b4/cmd/puppeth/wizard_genesis.go#L124

Rinkeby was also set up like this iirc.

Also, I believe that due to a bug in the spurious dragon hardfork, prefunding the precompiles is useful in some weird edge cases with different client / EVM implementations, see https://github.com/ethereum/EIPs/issues/716

Reply:
I've updated it to set the balance for the first 256 addresses